src/wp-admin/edit-comments.php

```diff
@@ -268,9 +292,9 @@
 		}
 
 		if ( $trashed > 0 ) {
-			$ids = isset($_REQUEST['ids']) ? $_REQUEST['ids'] : 0;
+			$ids = isset( $_REQUEST['ids'] ) ? $_REQUEST['ids'] : 0;
 			/* translators: %s: number of comments moved to the Trash */
-			$messages[] = sprintf( _n( '%s comment moved to the Trash.', '%s comments moved to the Trash.', $trashed ), $trashed ) . ' <a href="' . esc_url( wp_nonce_url( "edit-comments.php?doaction=undo&action=untrash&ids=$ids", "bulk-comments" ) ) . '">' . __('Undo') . '</a><br />';
+			$messages[] = sprintf( _n( '%s comment moved to the Trash.', '%s comments moved to the Trash.', $trashed ), $trashed ) . ' <a href="' . esc_url( wp_nonce_url( "edit-comments.php?doaction=undo&action=untrash&ids=$ids", 'bulk-comments' ) ) . '">' . __( 'Undo' ) . '</a><br />';
```

Comment:
Should this be expanded?

Reply:
We can see, it is a long line, matches upstream but it's a long concatenation - suggest for discussion.
https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/edit-comments.php

Reply:
As per review meeting, this is fine to keep the change